src/views/Home/Home.scss

```diff
@@ -6,8 +6,3 @@
     flex-wrap: wrap;
   }
 }
-
-.Button {
-  margin-right: var(--large-spacer);
-  margin-bottom: var(--medium-spacer);
-}
```

Comment:
I disagree with this change and think spacing should be relative to components in a view rather than the component itself.

What if we need a button that doesn't need this spacing? We'll need to undo or move the predefined spacing.

Reply:
Gotcha! By adding the styling to the `.Button` prop, we are essentially doing what I did. I will give the Button a classname within the `Home` view!